base/abstractarray.jl

```diff
@@ -2403,7 +2403,14 @@ function hash(A::AbstractArray, h::UInt)
     # Instead hash the tuple of firsts and lasts along each dimension
     h = hash(map(first, axes(A)), h)
     h = hash(map(last, axes(A)), h)
-    isempty(A) && return h
+
+    # For short arrays, it's not worth doing anything complicated
+    if length(A) < 8192
+        @inbounds for x in A
```

Comment:
```suggestion
        for x in A
```
This annotation seems like nonsense here?

(style nit: there should usually also be a space after a comma, on the next line)